src/models/thelio-mega-r4-n3/repairs.md

```diff
@@ -0,0 +1,700 @@
+# Thelio Mega (Parts & Repairs)
+
+Many components in your Thelio Mega can be upgraded or replaced as necessary. This page uses photos of the the B4-N3 revision, which indicates:
+
+- **B4:** The fourth Intel motherboard used in Thelio Mega.
+- **N3:** Based on the third revision of the nebula49 chassis.
+
+Minor case details may vary based on the production date of the unit, but screw counts, general component locations, and other details should remain the same unless otherwise noted.
+
+Power the machine off, switch off the power supply, and unplug all peripherals before working with any internal components. Then, follow these step-by-step guides for instructions:
+
+- [Replacing the front accent strip](#replacing-the-front-accent-strip)
+- [Removing the top case](#removing-the-top-case)
+- [Removing the side fan mount](#removing-the-side-fan-mount)
+- [Replacing the side intake fans](#replacing-the-side-intake-fans)
+- [Adding/removing 2.5" storage drives](#addingremoving-25-storage-drives)
+- [Removing the GPU brace](#removing-the-gpu-brace)
+- [Removing the CPU duct](#removing-the-cpu-duct)
+- [Replacing a GPU](#replacing-a-gpu)
+- [Replacing the M.2 drives](#replacing-the-m2-drives)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the CPU fans](#replacing-the-cpu-fans)
+- [Replacing the CPU cooler/thermal paste and CPU](#replacing-the-cpu-cooler-and-cpu)
+- [Replacing the RAM](#replacing-the-ram)
+- [Opening the PSU chamber](#opening-the-psu-chamber)
+- [Replacing the bottom intake fan](#replacing-the-bottom-intake-fan)
+- [Replacing the power supply](#replacing-the-power-supply)
+- [Removing the top crossbar](#removing-the-top-crossbar)
+- [Replacing the Thelio Io board](#replacing-the-thelio-io-board)
+- [Replacing the SATA backplanes](#replacing-the-sata-backplanes)
+- [Replacing the top I/O](#replacing-the-top-io)
+- [Troubleshooting the power button](#troubleshooting-the-power-button)
+
+## Replacing the front accent strip:
+
+Thelio Mira includes a customizable accent panel on the front of the case, which can be swapped to change the case's look and feel. The instructions for swapping the accent are also available in [video form](https://system76.com/r/diyaccentpanel).
```

Comment:
s/Mira/Mega/g

Reply:
Fixed all three occurrences of this in the file.